routes/addExpenses.js

```
@@ -0,0 +1,70 @@
+const pool = require('../models/db');
+const verifyToken = require('../verifyToken');
+
+const addExpenses = (app) => {
+    // Add expense
+    app.post('/:username/create_expense', verifyToken.verifyToken, (request, response) => {
+        const { username } = request.params;
+        const expense = {
+            created_by: username,
+            name: request.body.name,
+            description: request.body.description,
+            price: request.body.price,
+            created_at: request.body.created_at,
+            category: request.body.category,
+        };
+        pool.query('SELECT name, created_by FROM categories WHERE name = ? AND created_by = ?', [expense.category, username], (error, res) => {
+            if (error) {
+                response.send({
+                    code: 503,
+                    failed: 'Error occured',
```

Instead of just saying error occured, It would be better to return an error message describing what happened